### PR TITLE
Fix code and initial text early in the chapter for scheduling frames

### LINKED ISSUE #1358: My poor understanding about some sentence in scheduling.md
https://github.com/browserengineering/book/blob/567507769945691c3aa00527328a09a93553c3a5/book/scheduling.md?plain=1#L614-L617

Could you let me know whether I understand this sentence correctly.

Note how every time a frame is scheduled, we set up a timer to schedule the next
one.
-> Note that when `schedule_animation_frame` is called, we set up a timer to schedule the next `render`.

As a non-native English reader, I am confused by the two uses of the verb 'schedule.'

### PR DESCRIPTION
**Pull request by chrishtr**
The code was wrong, since there was nothing stopping animation frames timers from piling up and not being sequential.
Further, the text was confusing.

Fixes #1358.

### CODE REVIEW

File: book/scheduling.md [hunk 1]
```diff
@@ -1503,7 +1505,8 @@ class Tab:
         self.js.interp.evaljs("__runRAFHandlers()")
         self.render()
         commit_data = CommitData(
-            self.url, self.scroll, document_height, self.display_list)
+            self.url, self.scroll, document_height, \
+            self.display_list)
```
Comment (pavpanchekha) [inline book/scheduling.md:1509]: Does this matter? Does it go too long?

**Review by pavpanchekha — APPROVED** [review]
Good find / fix, sorry for sitting on it so long, today's my final

File: book/scheduling.md (hunk 1 above)
Comment (pavpanchekha) [reply book/scheduling.md:1509]: In either case feel free to merge.

Comment (chrishtr) [reply book/scheduling.md:1509]: Yeah it's because it caused horizontal scrollbars on the website.

**Pull request merged**